src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -4,45 +4,51 @@
    [status-im.chat.models.pin-message :as models.pin-message]
    [status-im.i18n.i18n :as i18n]
    [status-im.constants :as constants]
-   [status-im.utils.handlers :refer [<sub >evt]]
+   [status-im.utils.re-frame :as rf]
    [quo2.components.drawers.action-drawers :as drawer]))
 
-(defn- entry [icon label on-press danger?]
```

Comment:
since this is an action drawer maybe you could just call it `action` instead of `entry` although not that much important its not that much straightforward what an entry is

Reply:
The word `action` is being used to define the on-press method names. So, for example, we have `edit-nickname-entry` and `edit-nickname-action`